### DISCUSSION TITLE
can't see project changes to an issue in its history

### DISCUSSION THREAD

**Discussion by freddyrios** [community/community#7880] [Projects and Issues]
Example scenario: you have a shared board that multiple people can update. Someone has moved the priority levels for the issues in the board, but I don't see that information in the issue. Is it somewhere else?

**Comment by mestir** [comment]
osama osamaa vous a souri

**Comment by mestir** [comment]
.........................................................................................................................................
Join online meeting<https://join.skype.com/q9E3JccGQf0n>
.........................................................................................................................................
